### PR TITLE
docs: update the description of session lifetime in openid-connect.md

### PR DESCRIPTION
**Pull request by illidan33**
### Description

Documentation supplement for [10797](https://github.com/apache/apisix/pull/10919).
Modified the description of the new configuration in the document.
Remove any descriptions in the document that are not relevant to this update.

### CODE REVIEW

**Comment by monkeyDluffy6017** [comment]
please resolve the conflicts

**Comment by illidan33** [comment]
> please resolve the conflicts

Sure, already resolved.

File: docs/en/latest/plugins/openid-connect.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ description: OpenID Connect allows the client to obtain user information from th
...
 | session.cookie                       | object   | False    |                       |             |                                                                                                                                                                                  |
-| session.cookie.lifetime              | integer   | False    | 3600                  |             | it holds the cookie lifetime in seconds in the future. By default this is set to 3,600 seconds. This can be configured with Nginx set $session_cookie_lifetime 3600;. This does not set cookie's expiration time on session only (by default) cookies, but it is used if the cookies are configured persistent with session.cookie.persistent == true. |
+| session.cookie.lifetime              | integer   | False    | 3600                  |             | it holds the cookie lifetime in seconds in the future. By default this is set to 3,600 seconds. |
```
Comment (kayx23) [inline docs/en/latest/plugins/openid-connect.md:65]:
```suggestion
| session.cookie.lifetime              | integer   | False    | 3600                  |             | Cookie lifetime in seconds. |
```

File: docs/zh/latest/plugins/openid-connect.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ description: OpenID Connect（OIDC）是基于 OAuth 2.0 的身份认证协议
...
 | session.secret                       | string  | 是     | 自动生成               | 16 个以上字符  | 用于 session 加密和 HMAC 计算的密钥。 |
 | session.cookie                       | object   | False    |                       |             |                                                                                                                                                                                                                                                                                                                                 |
-| session.cookie.lifetime              | integer   | False    | 3600                  |             | 用于设置 cookie 的生命周期，以秒为单位。默认设置为 3600 秒。这可以通过 Nginx `set $session_cookie_lifetime 3600` 来配置。在默认情况下不会影响会话 cookie 的过期时间，但如果通过 session.cookie.persistent == true 配置了持久化的 cookie，则会使用该值。另请参阅有关 ssl_session_timeout 的说明。                                                                                                               |
+| session.cookie.lifetime              | integer   | False    | 3600                  |             | 用于设置 cookie 的生命周期，以秒为单位。默认设置为 3600 秒。  |
```
Comment (kayx23) [inline docs/zh/latest/plugins/openid-connect.md:64]:
```suggestion
| session.cookie.lifetime              | integer   | False    | 3600                  |             | 用于设置 cookie 的生命周期，以秒为单位。  |
```

File: docs/en/latest/plugins/openid-connect.md (hunk 1 above)
Comment (kayx23) [reply docs/en/latest/plugins/openid-connect.md:65]: No need to specify again the default lifetime in second, as it's already specified in the other column.

**Comment by illidan33** [comment]
@kayx23 @monkeyDluffy6017 Hi, docs have been updated.

**Review by kayx23 — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Pull request merged**